.github/workflows/e2e.yml

```diff
@@ -0,0 +1,46 @@
+name: e2e
```

Comment:
Intentionally separated this from the `ci` workflow, so they can run in parallel, which will hopefully take less time.

Reply:
this could go in the `react` workflow. It would run in parallel too :)

Reply:
Is there any harm in having it as a separate parallel workflow? It'll probably be a little longer-running as we add more tests, so nice to have it run separately. Also gives us the flexibility to have slightly different `on` triggers (e.g. excluding it for snapshots).